doc/chips/11.rst

```diff
@@ -42,38 +41,30 @@ separated by the proposed "finalize" keyword.
      ... Phase 2 code ...
    }
 
-Certain pieces of this syntax could be omitted as follows:
+We considered allowing certain pieces of this syntax to be omitted
+in some cases:
 
 * The super.init() call, if it takes no arguments.
-
-* The Phase 1 block statement, if contains no statements.
-  In this case, the "finalize" keyword can also be omitted.
-  Default field initializations followed by super.init() call
-  with no arguments occur in this case.
-
-Here is an example of an initializer where only Phase 2 is
-specified explicitly:
-
-.. code-block:: chapel
-
-   proc init() {
-     ... Phase 2 code ...
-   }
+* One of the two block statements.
+* The "finalize" keyword.
```

Comment:
Again, my memory of this conversation is that our discussion tended towards dropping the finalize block as a whole, not the keyword specifically.  Did you consult with our other team members on this change?  If not and if you continue to feel differently, I would like input from Mike or Michael to back up one position or the other, rather than leaving vague words that could be interpreted either way.